packages/app-store/components.tsx

```diff
@@ -20,10 +21,21 @@ export const InstallAppButtonWithoutPlanCheck = (
     type: App["type"];
   } & InstallAppButtonProps
 ) => {
+  const mutation = useAddAppMutation(null);
   const key = deriveAppDictKeyFromType(props.type, InstallAppButtonMap);
   const InstallAppButtonComponent = InstallAppButtonMap[key as keyof typeof InstallAppButtonMap];
   if (!InstallAppButtonComponent)
-    return <>{props.render({ useDefaultComponent: true, disabled: props.disableInstall })}</>;
```

Comment:
Was setting the disabled=false (always) intentional?

Reply:
Was testing, returned to original.